client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/list-item.js

```diff
@@ -22,11 +22,11 @@ import { errorNotice as errorNoticeAction, successNotice as successNoticeAction
 import { getSelectedSiteId } from 'state/ui/selectors';
 
 const CarrierAccountListItem = ( props ) => {
-	const { data, translate, errorNotice, successNotice, siteId } = props;
+	const { accountData, translate, errorNotice, successNotice, siteId } = props;
 
 	const [isDisconnectDialogVisible, setIsDisconnectDialogVisible] = React.useState(false);
 	const [isSaving, setIsSaving] = React.useState(false);
-	const [carrierId, setCarrierId] = React.useState(data.id);
+	const [carrierId, setCarrierId] = React.useState(accountData.id);
```

Comment:
:information_source: Missing spaces inside parens and brackets.

Reply:
I didn't change the formatting of this file - looks like it can be fixed by https://github.com/Automattic/woocommerce-services/issues/2097